judge/models/problem.py

```diff
@@ -108,6 +109,18 @@ class SubmissionSourceAccess:
     FOLLOW = 'F'
 
 
+class VotePermission(IntEnum):
+    NONE = 0
+    VIEW = 1
+    EDIT = 2
```

Comment:
Nit: Since this is not an enum general enough for other use cases, let's do a better job on naming, maybe:
- None (fine as is)
- View (fine, maybe comment on definition that this means "can view stats")
- Vote (Edit sounds like you can edit votes, which is I guess true, but I think `.can_vote()` is a much better method name.

Reply:
done